tests/cases/conformance/jsdoc/jsdocAbstract2.ts

```diff
@@ -0,0 +1,20 @@
+// @allowJs: true
+// @checkJs: true
+// @target: esnext
+// @noEmit: true
```

Comment:
can you change this to `@emitDeclarationOnly: true` to also test the declaration emit?